recipe/meta.yaml

```diff
@@ -23,13 +23,13 @@ outputs:
         - pip
         - setuptools
         - wheel
-        - packaging 20.0
-        - typing-extensions 4.4.0
+        - packaging >=20.0
+        - typing-extensions
       run:
         - python
         - importlib-metadata  # [py<38]
         - packaging >=20.0
-        - setuptools >=45
+        - setuptools
         - tomli >=1.0.0  # [py<311]
         - typing-extensions
```

Comment:
```suggestion
        - typing-extensions  # [py<311]
```

Reply:
or this one https://github.com/pypa/setuptools_scm/blob/v7.1.0/setup.cfg#L33